data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
+		payment 500000 6
+
+mission "Wanderers: Spera Anatrusk Outfitter 3c"
+	landing
+	name "Technicans to <planet>"
+	description "Bring repair technicians from <stopovers> to <destination>. Payment is <payment>."
+	blocked "You are [lacking, missing] <capacity> to transport these repair technicians."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Var' Kayi"
+	passengers 30
+	cargo "tools" 200
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 3: offered"
+
+	on stopover
+		dialog "The repair techs seem more concerned about making sure that their tools are securely stored than about speculating on the Eye, though you do have to chase a few out of the engineering section as they poke around to see how the <ship> is outfitted."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort with the repair technicians hasn't landed. You realize you need to depart and make sure they arrive safely."
+
+	on abort
+		event "wanderers: spera anatrusk colony" 30
+
+	on fail
+		event "wanderers: spera anatrusk colony" 30
+
+	on complete
+		dialog "The repair techs check that their tools are intact and then walk over to the warehouse. It looks like they will be working outside until the new repair hangars are built."
+		payment 300000 9
+
+
+mission "Wanderers: Spera Anatrusk Outfitter 4"
+	landing
+	name "Escort outfitter convoy"
+	description "Escort outfitter convoy to <destination>."
+	source "Spera Anatrusk"
+	destination "Spera Anatrusk"
+	stopover "Vara K'chrai"
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 3a: done"
+		has "Wanderers: Spera Anatrusk Outfitter 3b: done"
+		has "Wanderers: Spera Anatrusk Outfitter 3c: done"
+
+	on offer
+		conversation
+			`You watch the work crews unload the cargo by uncrating the repulsor lifts in the hold. They then use them to move the crates of shelving and supplies into the prefabricated shell that stands on what was formerly an empty field. The spaceport tent has been moved and you can see a new line of stakes where the spaceport building itself will be.`
+			`	Clearly other supplies are coming in as well; you can see three Deep Rivers being unloaded and containers of equipment and construction materials being stacked in a massive yard near the outfitter building.`
+			`	The construction supervisor is shouting into some kind of bull horn to one of the work crews, trying to be heard over the heavy equipment. When he notices you, he stops and flutter-walks over to you.`
+			`"Captain! It is [exciting, nerve-wracking] to see that the equipment is here. We have a [fleet, convoy] of freighters to be loaded with outfits at <stopovers>. If you can escort them, we will have everything we need to open at least a minimal outfitter."`
+			
+			choice
+				`	"I'd be glad to."`
+					accept
```

Comment:
Excuse me if I'm wrong, but won't this end the coversation?

Reply:
Good catch. Thanks. Fixed.